README.md

```diff
@@ -128,8 +128,8 @@ metadata:
   name: cluster
 spec:
   kubeMacPool:
-   startPoolRange: "02:00:00:00:00:00"
-   endPoolRange: "FD:FF:FF:FF:FF:FF"
+   startRange: "02:00:00:00:00:00"
```

Comment:
Maybe I misunderstood Dan's comment, but I thought the request was to change it to "rangeStart"/"rangeStop". @dankenigsberg?

Reply:
> I'm not a native English speaker, but I think 'StartPoolRange' is not proper. I'd prefer to call it 'RangeStart'; it is quite clear that it is an attribute of the Pool.

Reply:
you are right. my mistake

Reply:
I think that rangeStart/rangeEnd is better English.